tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeGhQuantities.cpp

```diff
@@ -102,6 +102,31 @@ void test_compute_extrinsic_curvature_and_deriv_metric(const T& used_for_size) {
   CHECK_ITERABLE_APPROX(deriv_spatial_metric, deriv_spatial_metric_test);
 }
 
+
+template <typename DataType, size_t SpatialDim, typename Frame>
+void test_3p1_deriv_functions(const DataVector& used_for_size) noexcept {
+  // spatial_deriv_of_lapse
+  pypp::check_with_random_values<1>(
+      static_cast<tnsr::i<DataType, SpatialDim, Frame> (*)(
+        const Scalar<DataType>&,
+        const tnsr::A<DataType, SpatialDim, Frame>&,
+        const tnsr::iaa<DataType, SpatialDim, Frame>&)>(
+          &::GeneralizedHarmonic::spatial_deriv_of_lapse<SpatialDim, Frame,
+          DataType>), "TestFunctions", "deriv_lapse",
+      {{{std::numeric_limits<double>::denorm_min(), 10.0}}}, used_for_size);
+  // time_deriv_of_lapse
+  pypp::check_with_random_values<1>(
+      static_cast<Scalar<DataType> (*)(
+        const Scalar<DataType>&,
+        const tnsr::I<DataType, SpatialDim, Frame>&,
+        const tnsr::A<DataType, SpatialDim, Frame>&,
+        const tnsr::iaa<DataType, SpatialDim, Frame>&,
+        const tnsr::aa<DataType, SpatialDim, Frame>&)>(
+      &::GeneralizedHarmonic::time_deriv_of_lapse<SpatialDim, Frame, DataType>),
+      "TestFunctions", "dt_lapse",
+      {{{std::numeric_limits<double>::denorm_min(), 10.0}}}, used_for_size);
```

Comment:
Could you verify that you get the right answer for Kerr-Schild?

Reply:
Just added KerrSchild test.